src/convertToObject.js

```diff
@@ -1,18 +1,25 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
```

Comment:
Pls see checklist point 2
[CODE STYLE] - Avoid variable namings like `array` , `object`, `arr`, `obj`, `result`. It will decrease code readability

Reply:
My bad, that was so common to use obj naming that i didnt pay atterntion to that part in checklist. Will improve it